system-tests/src/tests/course-material/content-components-render.spec.ts

```diff
@@ -0,0 +1,46 @@
+import { test } from "@playwright/test"
+
+import expectScreenshotsToMatchSnapshots from "../../utils/screenshot"
+test.use({
+  storageState: "src/states/admin@example.com.json",
+})
+test("blocks render correctly", async ({ page, headless }) => {
+  // Go to http://project-331.local/
+  await page.goto("http://project-331.local/")
+  // Click [id="__next"] div >> :nth-match(div:has-text("University of Helsinki, Department of Computer ScienceOrganization for Computer "), 4)
+  await Promise.all([
+    page.waitForNavigation(),
+    page.click(
+      '[id="__next"] div >> :nth-match(div:has-text("University of Helsinki, Department of Computer ScienceOrganization for Computer "), 4)',
+    ),
+  ])
+  // Click text=Introduction to Course Material
+  await Promise.all([
+    page.waitForNavigation(/*{ url: 'http://project-331.local/courses/introduction-to-course-material' }*/),
+    page.click("text=Introduction to Course Material"),
+  ])
+  // Click button:has-text("Continue")
+  await page.click('button:has-text("Continue")')
+  // Click text=User Experience
+  await Promise.all([
+    page.waitForNavigation(/*{ url: 'http://project-331.local/courses/introduction-to-course-material/chapter-2' }*/),
+    page.click("text=User Experience"),
+  ])
+  // Click text=Content rendering
+  await Promise.all([
+    page.waitForNavigation(/*{ url: 'http://project-331.local/courses/introduction-to-course-material/chapter-2/content-rendering' }*/),
+    page.click("text=Content rendering"),
+  ])
+
+  await page.waitForSelector("text=100px wide")
+
+  await expectScreenshotsToMatchSnapshots({
+    page,
+    headless,
+    snapshotName: "content-components-renderer-view",
+    waitForThisToBeVisibleAndStable: null,
+    toMatchSnapshotOptions: { threshold: 0.4 },
+    pageScreenshotOptions: { fullPage: true },
+    axeSkip: true,
```

Comment:
Resolve

Reply:
Wil lresolve in another task as Background colors for headings color palette is disabled for some reason on the latest version.